spec/features/idv/doc_capture/document_capture_step_spec.rb

```diff
@@ -145,20 +147,18 @@
       )
     end
 
-    context 'when javascript is enabled', :js do
-      it 'logs return to sp link click' do
-        complete_doc_capture_steps_before_first_step(user)
-        new_window = window_opened_by do
-          click_on t('idv.troubleshooting.options.get_help_at_sp', sp_name: sp_name)
-        end
-
-        within_window new_window do
-          expect(fake_analytics).to have_logged_event(
-            'Return to SP: Failed to proof',
-            step: 'document_capture',
-            location: 'document_capture_troubleshooting_options',
-          )
-        end
+    it 'logs return to sp link click' do
+      complete_doc_capture_steps_before_first_step(user)
+      new_window = window_opened_by do
+        click_on t('idv.troubleshooting.options.get_help_at_sp', sp_name: sp_name)
+      end
+
+      within_window new_window do
+        expect(fake_analytics).to have_logged_event(
+          'Return to SP: Failed to proof',
+          step: 'document_capture',
+          location: 'document_capture_troubleshooting_options',
+        )
       end
```

Comment:
`fake_analytics` shouldn't be per-window right? it's on the shared server? do we need `within_window` here?

Reply:
Well, I can't explain why, but it does seem to need to be this way, and fails otherwise 🤷